napari_tiff/napari_tiff_reader.py

```diff
@@ -72,14 +70,30 @@ def zip_reader(path: PathLike) -> List[LayerData]:
 
 
 def tifffile_reader(tif):
-    """Return napari LayerData from largest image series in TIFF file."""
+    """Return napari LayerData from image series in TIFF file."""
+    nlevels = len(tif.series[0])
```

Comment:
I'm still a little confused about the difference between tiff series and tiff pages. Is anyone else able to explain it?

Reply:
Although @cgohlke is the expert as creator of tifffile, as I understand it pages is the internal structure of tiff, and the series/levels provide higher level access (which point to the underlying pages).